Revit_Core_Engine/Revit_Core_Engine.csproj

```diff
@@ -88,7 +88,7 @@
     <ErrorReport>prompt</ErrorReport>
     <CodeAnalysisRuleSet>ManagedMinimumRules.ruleset</CodeAnalysisRuleSet>
     <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
-    <AssemblyName>Revit_Core_Engine_2020</AssemblyName>
+    <AssemblyName>Revit_Core_Engine</AssemblyName>
```

Comment:
@pawelbaran is there a risk associated with this change?

Reply:
Well spotted, my mistake, will commit soon with change back to 2020